modules/aruco/src/apriltag_quad_thresh.cpp

```diff
@@ -0,0 +1,1600 @@
+/* Copyright (C) 2013-2016, The Regents of The University of Michigan.
+All rights reserved.
+
+This software was developed in the APRIL Robotics Lab under the
+direction of Edwin Olson, ebolson@umich.edu. This software may be
+available under alternative licensing terms; contact the address above.
```

Comment:
@szk1509 Could you ask original authors about using of OpenCV license for this code?

We planning to move aruco module into the main repository in OpenCV 4.0, that requires clean code as much as possible from legal point of view.

Reply:
we got the permission, here is the reply by Edwin Olson:
> Hi Pavel,
>
> The 3 clause BSD license is also acceptable. 
>
> Looking forward to seeing this in OpenCV!
>
> Best,
>
> -Ed

Reply:
@paroj @edwinolson Thanks!